test/integration/features/basic.feature

```diff
@@ -31,7 +31,7 @@ Feature: Basic test
         When executing "crc status" fails
         Then stderr should contain 
         """
-        Machine "crc" does not exist. Use "crc start" to add a new one.
+        Virtual machine 'crc' does not exist, use 'crc start' to create it.
```

Comment:
We prefer not to actually talk about Virtual machine, but instance or Machine. The use of the VM terminilogy has been seen by outsiders as daunting and an obstacle for adoption. (while a VM is actually used).

Reply:
There're several more places in the code that mention a 'CRC VM':
```
$ ack 'CRC VM'
pkg/crc/machine/machine.go
251:		return *result, errors.New("Failed to connect to the CRC VM with SSH")

cmd/crc/cmd/setup.go
18:		machine.DefaultDriver.Driver, fmt.Sprintf("The driver to use for the CRC VM. Possible values: %v", machine.SupportedDriverValues()))

cmd/crc/cmd/delete.go
43:	yes := input.PromptUserForYesOrNo("Do you want to delete the CRC VM?", globalForce)

cmd/crc/cmd/start.go
77:	flagSet.StringP(config.VMDriver.Name, "d", machine.DefaultDriver.Driver, fmt.Sprintf("The driver to use for the CRC VM. Possible values: %v", machine.SupportedDriverValues()))
79:	flagSet.IntP(config.CPUs.Name, "c", constants.DefaultCPUs, "Number of CPU cores to allocate to the CRC VM")
80:	flagSet.IntP(config.Memory.Name, "m", constants.DefaultMemory, "MiB of memory to allocate to the CRC VM")

cmd/crc/cmd/status.go
23:	Long:  "Show details about OpenShift cluster and the CRC VM",
29:var statusFormat = `CRC VM:          {{.CrcStatus}}
67:	diskUsage := fmt.Sprintf("%s of %s (Inside the CRC VM)", diskUse, diskSize)
$
```
Do we want to rephrease these, too?

Reply:
We need to consider a consistent option... either we use VM (and break this 'taboo' as existed with Minishift, or we use the full 'Virtual machine'. @code-ready/crc-devel WDYT?